layouts/partials/search.html

```diff
@@ -0,0 +1,19 @@
+<section class="mb-3">
+    <div class="md:flex md:justify-between">
+        <div id="playground-search-form" class="mb-6 md:mb-0 md:ml-2 flex gap-1.5">
+            <input type="search" id="playground-search-term" placeholder="Enter search term" autocomplete="off" class="bg-[#D9D9D9] border border-black w-80 pl-2" />
+            <button id="search-submit">
+                <img src="/img/search-icon.svg" alt="" />
+            </button>
```

Comment:
maybe display the search bar if  $data.playground. searchbar is enabled?  you could set it up in `data/en/ `folder

```yml path=/data/en/playground
searchbar:
  enable:true
filter:
  enable:true


```

Reply:
I added `is_search_enabled` and `is_filtered_enabled` to the `/data-playground/_index.md` file.